flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/operators/join/SortMergeJoinOperator.java

```diff
@@ -98,32 +97,15 @@
 	private transient BaseRow rightNullRow;
 	private transient JoinedRow joinedRow;
 
-	@VisibleForTesting
 	public SortMergeJoinOperator(
-			long reservedSortMemory, long externalBufferMemory, FlinkJoinType type, boolean leftIsSmaller,
+			double leftMemRatio, long externalBufferMemory, FlinkJoinType type, boolean leftIsSmaller,
```

Comment:
This seems adhoc now, we can simply split the remaining memory evenly for both sorters

Reply:
OK, let's remove legacy codes.